SFDigitalServices/sf-dahlia-web
# feat: DAH-2274 GoogleTranslationService

DAH-2274

This PR includes the GoogleTranslationService, which is responsible for calling the Google Cloud Translate API

Testing Instructions
- Open Heroku and go to this PR env: https://dashboard.heroku.com/apps/dahlia-webapp-pr-2033
- More -> Run Console -> type in `bash`
- Once it is ready, navigate to `/lib/tasks`
- Run `rake translate_test:translate`
- You should see this output: 
`Connecting to Google Cloud Translate...
Test translation results [{:to=>"ES", :translation=>["Hola", "mundo"]}, {:to=>"ZH", :translation=>["你好", "世界"]}, {:to=>"TL", :translation=>["Kamusta", "mundo"]}]`

You can also test this locally. Ensure you run `yarn install` and `bundle install` and add the GOOGLE_TRANSLATE_KEY  and GOOGLE_PROJECT_ID to you .env. You can find the value in the heroku environment for this PR.
